eldritch/test_events.py

```diff
@@ -88,7 +88,7 @@ def resolve_until_done(self):
       not_finished["stack"] = self.state.event_stack
     self.assertFalse(not_finished)
 
-  def resolve_to_choice(self, event_class):
+  def resolve_to_choice(self, event_class) -> events.ChoiceEvent:
```

Comment:
This mainly helps my IDE warn me when I try to do `choice.resolve("thing")` instead of `choice.resolve(self.state, "thing")`